### PR TITLE
RHDEVDOCS-6115: Content creation for 1.11.6 RN

### PR DESCRIPTION
**Pull request by Dhruv-Soni11**
Version(s):

GitOps 1.11, GitOps 1.12, GitOps 1.13

Issue:

https://issues.redhat.com//browse/RHDEVDOCS-6115

Link to docs preview:

[GitOps 1.11.6 release notes](https://78988--ocpdocs-pr.netlify.app/openshift-gitops/latest/release_notes/gitops-release-notes.html#release-notes-for-gitops-1-11-6_gitops-release-notes)

QE review:
- [x] QE has approved this change.

SME review: anjoseph@redhat.com [lzhan@redhat.com](mailto:lzhan@redhat.com) [pmurkute@redhat.com](mailto:pmurkute@redhat.com) kabhishe@redhat.com
QE review: @varshab1210 
Peer review: @Srivaralakshmi 

Additional information:

### CODE REVIEW

**Review by anandf — APPROVED** [review]
LGTM

**Comment by varshab1210** [comment]
LGTM

**Comment by Dhruv-Soni11** [comment]
This PR contains incomplete errata information as the errata is not yet published. I will update the errata details once they are ready.

File: modules/gitops-release-notes-1-11-6.adoc [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+[id="release-notes-for-gitops-1-11-6_{context}"]
+= Release Notes for {gitops-title} 1.11.6
+
+{gitops-title} 1.11.6 is now available on {OCP} 4.12, 4.13, and 4.14.
+
+[id="errata-updates-1-11-6_{context}"]
+== Errata updates
+
+[id="rhsa-2024-133429-09-gitops-1-11-6-security-update-advisory_{context}"]
+=== RHSA-2024:133429-09 - {gitops-title} 1.11.6 security update advisory
```
Comment (Srivaralakshmi) [inline modules/gitops-release-notes-1-11-6.adoc:14]:
```suggestion
=== RHSA-2024:133429 - {gitops-title} 1.11.6 security update advisory
```

File: modules/gitops-release-notes-1-11-6.adoc [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+Issued: 2024-07-18
+
+The list of security fixes that are included in this release is documented in the following advisory:
+
+* link:https://access.redhat.com/errata/RHSA-2024:133429-09[RHSA-2024:133429-09]
```
Comment (Srivaralakshmi) [inline modules/gitops-release-notes-1-11-6.adoc:20]:
```suggestion
* link:https://access.redhat.com/errata/RHSA-2024:133429[RHSA-2024:133429]
```

File: release_notes/gitops-release-notes.adoc [hunk 3]
```diff
@@ -7,6 +7,13 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
+[NOTE]
+====
+For additional information about the OpenShift GitOps life cycle and supported platforms, refer to the link:https://access.redhat.com/support/policy/updates/openshift_operators[OpenShift Operator Life Cycles] and link:https://access.redhat.com/support/policy/updates/openshift[Red Hat OpenShift Container Platform Life Cycle Policy].
```
Comment (Srivaralakshmi) [inline release_notes/gitops-release-notes.adoc:12]:
```suggestion
For additional information about the OpenShift {gitops-shortname} life cycle and supported platforms, refer to the link:https://access.redhat.com/support/policy/updates/openshift_operators[OpenShift Operator Life Cycles] and link:https://access.redhat.com/support/policy/updates/openshift[Red{nbsp}Hat {OCP} Life Cycle Policy].
```

Comment (Srivaralakshmi) [reply release_notes/gitops-release-notes.adoc:12]: @Dhruv-Soni11 In the go-compatibility-and-support-matrix.adoc file, please add this information under the IMPORTANT admonition  as a second bullet:

The maintenance support for {OCP} 4.12 on {ibmpowerProductName}, {ibmzProductName}, and {linuxoneProductName} has ended from 17 July 2024. If you are using {gitops-title} on {OCP} 4.12, upgrade to {OCP} 4.13 or later.

**Review by Srivaralakshmi — COMMENTED** [review]
@Dhruv-Soni11 I have left some comments. PTAL and implement them. Thank you!

File: modules/gitops-release-notes-1-11-6.adoc [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+:_mod-docs-content-type: REFERENCE
+[id="release-notes-for-gitops-1-11-6_{context}"]
+= Release Notes for {gitops-title} 1.11.6
+
+{gitops-title} 1.11.6 is now available on {OCP} 4.12, 4.13, and 4.14.
+
+[id="errata-updates-1-11-6_{context}"]
+== Errata updates
+
+[id="rhsa-2024-133429-09-gitops-1-11-6-security-update-advisory_{context}"]
```
[the remark this answers is not in the document]
Comment (Dhruv-Soni11) [reply modules/gitops-release-notes-1-11-6.adoc:13]: Fixed it now. Will resolve this comment only when the correct errata numbers are generated.

File: modules/gitops-release-notes-1-11-6.adoc (hunk 1 above)
Comment (Dhruv-Soni11) [reply modules/gitops-release-notes-1-11-6.adoc:14]: Fixed.

File: modules/gitops-release-notes-1-11-6.adoc (hunk 2 above)
Comment (Dhruv-Soni11) [reply modules/gitops-release-notes-1-11-6.adoc:20]: Fixed.

File: release_notes/gitops-release-notes.adoc (hunk 3 above)
Comment (Dhruv-Soni11) [reply release_notes/gitops-release-notes.adoc]: Fixed.

**Review by Srivaralakshmi — COMMENTED** [review]
@Dhruv-Soni11 LGTM. Thanks for the changes.

Please get the review and approval from the RH Partner-IBM team people involved in the [4.12 EOL slack chat](https://redhat-internal.slack.com/archives/C03GGC05NGG/p1721218333943469).

**Review by Srivaralakshmi — COMMENTED** [review]
@Dhruv-Soni11 LGTM. Thanks!

**Comment by Dhruv-Soni11** [comment]
SME, QE, and Peer reviews have been completed. I just need final merge review and CP to the branches mentioned in the description. Thanks.

**Review by Srivaralakshmi — COMMENTED** [review]
LGTM

**Pull request merged**